ecs/env.json

```diff
@@ -217,6 +217,14 @@
     {
       "name": "ONBOARDING_EMAIL_WEBHOOK_URL",
       "valueFrom": "plumber-onboarding-email-webhook-url"
+    },
+    {
+      "name": "AISAY_API_URL",
+      "valueFrom": "plumber-<ENVIRONMENT>-aisay-api-url"
+    },
+    {
+      "name": "AISAY_GET_TOKEN_URL",
+      "valueFrom": "plumber-<ENVIRONMENT>-aisay-get-token-url"
```

Comment:
as discussed, can consider putting this in a helper function in the `auth` folder to `getAPIUrl` and `getTokenUrl` separately so can avoid the trouble of storing extra URL config in the parameter store (esp since its app specific for AISAY)